onnxruntime/contrib_ops/cpu_contrib_kernels.cc

```diff
@@ -67,8 +67,8 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSNchwcDomai
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSNchwcDomain, 1, float, GlobalMaxPool);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSNchwcDomain, 1, float, AveragePool);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSNchwcDomain, 1, float, GlobalAveragePool);
-class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 1, float, LayerNormalization);
-class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 1, double, LayerNormalization);
```

Comment:
Scott McKay: 
Are there any models with opset < 9 that potentially use this op, or is this change just making the SinceVersion correct?
Sherlock Huang:
LayerNorm should actually by in kMsDomain, since it's not part of ONNX spec yet.
Currently, LayerNorm op is only introduced through layer_norm_fusion.
So, in theory, not model should be affected
We are updating the opset to 9, coz training only support onnx opset 9+.